chain-vote/src/lib.rs

```diff
@@ -121,6 +123,7 @@ impl Tally {
     }
 }
 
+#[allow(clippy::ptr_arg)]
```

Comment:
The last argument could be &[TallyDecryptShare], clippy is right, no need to pass a `Vec` by reference.

Reply:
I did not feel confident in changing the API. But It makes sense.